twinclient2/tfchain.v

```
@@ -0,0 +1,82 @@
+module twinclient2
+import json
+
+
+
+fn new_tfchain(mut client TwinClient) TfChain {
+	return TfChain{
+		client: unsafe {client}
+	}
+}
+
+pub fn (mut c TfChain) import_(name string, mnemonics string)?{
+	c.client.send(
+		'tfchain.import',
+		json.encode({"name": name, "mnemonics": mnemonics})
+	)?
+}
+
+pub fn (mut c TfChain) get(name string)? TfchainWalletAddressModel{
+	// Returns address as string.
+	response := c.client.send(
+		'tfchain.get',
+		json.encode({"name": name})
+	)?
+	return json.decode(
+		TfchainWalletAddressModel,
+		response.data
+	) or {}
```

we shouldn't suppress this error here and return an empty result

```suggestion
    )
```

Also, I think these lines can be written in a single line

```suggestion
    return json.decode(TfchainWalletAddressModel,response.data)
```